src/monattack.cpp

```diff
@@ -3522,6 +3522,13 @@ bool mattack::searchlight( monster *z )
     return true;
 }
 
+bool mattack::speaker( monster *z )
+{
+    sounds::sound( z->pos(), 60, sounds::sound_t::order, ( _( "%s" ),
+                   SNIPPET.random_from_category( "speaker_warning" ) ) );
```

Comment:
This is not doing what you think it does.

This is a pointless translation. How is a translator supposed to know what they should translate? Why even translate this?
```C++
_( "%s" )
```

This invokes the comma-operator, which evaluates both arguments (calls `_` and `random_from_category`) and returns the value of the right operand and discards the value of the left operand. So the translations (if any) is discarded anyway.
```C++
( _( "%s" ), SNIPPET.random_from_category( "speaker_warning" ) )
```
```suggestion
    sounds::sound( z->pos(), 60, sounds::sound_t::order, SNIPPET.random_from_category( "speaker_warning" ) );
```

(Note that `random_from_category` returns an already translated string.)